<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>911</width>
    <height>458</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="playBTN">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>360</y>
      <width>71</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Play/Pause</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>10</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Available Songs</string>
    </property>
   </widget>
   <widget class="QSlider" name="ProgressSlider">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>330</y>
      <width>621</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QListView" name="listView_playlist">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>30</y>
      <width>201</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="rewindBTN">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>360</y>
      <width>80</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Rewind</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addPlaylistBTN">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>220</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Song to Playlist</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ffBTN">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>360</y>
      <width>80</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Fast Forward</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>10</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Current Playlist</string>
    </property>
   </widget>
   <widget class="QPushButton" name="downloadBTN">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>250</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Download Song</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="songLineEdit">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>300</y>
      <width>541</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QSlider" name="volumeSlider">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>360</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>350</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../assets/speaker.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="uploadBTN">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>250</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Upload Song</string>
    </property>
   </widget>
   <widget class="QPushButton" name="skipBTN">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>360</y>
      <width>80</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Skip Track</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="TrackTime">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>300</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QListView" name="listView_availSongs">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>30</y>
      <width>201</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="serverPort">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Port Number</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="serverIP">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>IP Address</string>
    </property>
   </widget>
   <widget class="QPushButton" name="serverStartBTN">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>111</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Server Start</string>
    </property>
   </widget>
   <widget class="QListView" name="listView_playlist_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>211</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Connected Clients</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>911</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
